Donation-platform/src/main/webapp/personalized.html

```diff
@@ -6,29 +6,44 @@
     <script src="personalizationScript.js"></script>
   </head>
   <body>
-
     <div>
+      <!-- TODO: prevent user from selecting same tag multiple times (remove selected tag from following forms) -->
       <form name="tag-input" action="/personalize" method="POST">
-        <legend>Select the causes you most care about:</legend>
-        <div>
-          <input type="checkbox" id="blm" name="blm" value="Black Lives Matter">
-          <label for="blm">Black Lives Matter</label>
-        </div>
-        <div>
-          <input type="checkbox" id="education" name="education" value="Education">
-          <label for="education">Education</label>
-        </div>
-        <div>
-          <input type="checkbox" id="hunger-and-poverty" name="hunger-and-poverty" value="Hunger and Poverty">
-          <label for="hunger-and-poverty">Hunger and Poverty</label>
-        </div>
-        <div>
-          <input type="checkbox" id="environment" name="environment" value="Environmental Protection">
-          <label for="environment">Environmental Protection</label>
-        </div>
-        <input type="submit" value="Submit" />
+        <label for="tag-1">Select the #1 cause you care about:</label>
+        <select name="tag-1" id="tag-1" multiple>
+          <option value="hunger">hunger</option>
+          <option value="medicine">medicine</option>
+          <option value="education">education</option>
+          <option value="children">children</option>
+          <option value="environment">environment</option>
+          <option value="race">race</option>
+          <option value="health">health</option>
+        </select>
+        <br><br>
+        <label for="tag-2">Select the #2 cause you care about:</label>
+        <select name="tag-2" id="tag-2" multiple>
+          <option value="hunger">hunger</option>
+          <option value="medicine">medicine</option>
+          <option value="education">education</option>
+          <option value="children">children</option>
+          <option value="environment">environment</option>
+          <option value="race">race</option>
```

Comment:
Maybe change to racial equality/justice?

Reply:
Fixed with recent commit.